$NetBSD: patch-Headers_GNUstepBase_GSConfig.h.in,v 1.1 2015/05/08 14:16:21 christos Exp $

Avoid redefinition of macros from cdefs_elf.h; this is not the OS's fault,
but GNUstep's for violating the implementation namespace with __weak and
friends.

--- Headers/GNUstepBase/GSConfig.h.in.orig	2015-05-07 15:49:03.000000000 +0000
+++ Headers/GNUstepBase/GSConfig.h.in
@@ -386,6 +386,8 @@ typedef	struct {
 #endif
 
 #if !__has_feature(objc_arc)
+#  undef __weak
+#  undef __strong
 #  if __OBJC_GC__
 #    define __strong __attribute__((objc_gc(strong)))
 #    define __weak __attribute__((objc_gc(weak)))
